clearpath_hardware_interfaces/include/clearpath_hardware_interfaces/lighting/color.hpp

```diff
@@ -62,18 +62,18 @@ struct hsv_t {
   }
 };
 
-static const hsv_t COLOR_RED = hsv_t(0.0, 100.0, 100.0);
-static const hsv_t COLOR_RED_DIM = hsv_t(0.0, 100.0, 10.0);
-static const hsv_t COLOR_MAGENTA = hsv_t(300.0, 100.0, 100.0);
-static const hsv_t COLOR_BLUE = hsv_t(240.0, 100.0, 100.0);
-static const hsv_t COLOR_BLUE_DIM = hsv_t(240.0, 100.0, 10.0);
-static const hsv_t COLOR_CYAN = hsv_t(180.0, 100.0, 100.0);
-static const hsv_t COLOR_GREEN = hsv_t(120.0, 100.0, 100.0);
-static const hsv_t COLOR_GREEN_DIM = hsv_t(120.0, 100.0, 10.0);
-static const hsv_t COLOR_YELLOW = hsv_t(60.0, 100.0, 100.0);
-static const hsv_t COLOR_ORANGE = hsv_t(30.0, 100.0, 100.0);
-static const hsv_t COLOR_WHITE = hsv_t(0.0, 0.0, 100.0);
-static const hsv_t COLOR_WHITE_DIM = hsv_t(0.0, 0.0, 10.0);
+static const hsv_t COLOR_RED = hsv_t(0.0, 100.0, 37.5);
```

Comment:
We will need to test these values on other robots and adjust as necessary

Reply:
Is there a ticket somewhere to track that testing so it doesn't get lost/forgotten about?

Reply:
I'll make one

Reply:
Thanks!

Reply:
RPSW-2399